lesson14.md

```diff
@@ -0,0 +1,511 @@
+<!-- .slide: id="lesson14" -->
+
+# Basic Frontend - Fall 2020
+
+Lesson 14, Tuesday, 2020-10-13
+
+---
+
+Lesson overview:
+
+- functions recap
+  - how to call a function by clicking a button
+- introduction to the DOM API
+- interacting with HTML elements through the DOM API
+- changing colors and content through JavaScript
+
+---
+
+### Recap: Functions
+
+1. What are functions?
+1. Why do we need functions?
+1. What are functions useful for?
+
+---
+
+### Recap: Functions
+
+- a function is a reusable block of code
+- a function allows us to group together several lines of code that we can
+  easily run just by executing the function
+- functions are handy and convenient
+- functions let us structure our code better and let us avoid repetitions 
+
+---
+
+### Recap: Functions
+
+
+![Page](images/lesson-14-function-coffee.png) <!-- .element width="500px" style="display: block; margin: 0 auto;" -->
+
+---
+
+### Recap: functions
+
+To use a function:
+
+1. first we need to create, define, a function
+  - function keyword
+  - function name
+  - parantheses ()
+  - optional parameters
```

Comment:
no body?

Reply:
I figured that that's where most of the bugs usually are...